tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
it's confusing why we need ts, is it for sitemap generator package?

Reply:
I think it came with the sitemap generator package when it was installed

Reply:
Oh nvm, it was because the index for server-sitemap.xml was tsx so run build was autocreating a tsconfig. I've converted the tsx into jsx which should fix it